src/App.js

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quote from './components/Quote';
+import './css/nav.css';
 
 export default function App() {
   return (
-    <Calculator />
+    <>
+      <nav>
+        <h1 className="nav-bar-h1">Math Magicians</h1>
+        <ul className="nav-link">
+          <li><Link to="/">Home</Link></li>
+          <li><Link to="/calculator">Calculator</Link></li>
+          <li><Link to="/quote">Quote</Link></li>
```

Comment:
- Optional
I would strongly recommend using `NavLink` component instead of `Link` component.
I would also recommend using a separate functional component **Navbar** for the functionality of navbar that your implemented instead of including it in the **App.js**